core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -435,7 +435,7 @@ package object config {
         "a property key or value, the value is redacted from the environment UI and various logs " +
         "like YARN and event logs.")
       .regexConf
-      .createWithDefault("(?i)secret|password".r)
+      .createWithDefault("(?i)secret|password|oauthToken".r)
```

Comment:
Can we just have `token` instead of `oauthToken`?

Reply:
yep, will update